src/components/ItemGrid.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="row">
+    <app-item v-for="(item, index) in items" @click.native="deleteItem(index)">{{ item }}</app-item>
+  </div>
+</template>
+
+<script>
+import Item from "./Item.vue";
+
+export default {
+  props: ["items"],
```

Comment:
Props se dají psát takhle jednoduše. COž je dobrý tak pro tohle demo.
V produkci si to budeš chtít otypovat 
takhle to potom nějak vypadá 
![obrazek](https://user-images.githubusercontent.com/15363559/84014234-495a9980-a97a-11ea-81a9-d57c73b252dc.png)